Src/DependencyCollector/Shared.Tests/DependencyCollector.Shared.Tests.projitems

```diff
@@ -13,6 +13,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)HttpDependenciesParsingTelemetryInitializerTest.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\ClientServerDependencyTrackerTests.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\DependencyTargetNameHelperTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Implementation\HttpParsers\HttpParsingHelperTests.cs" />
```

Comment:
Please add unit tests on the new parser with the specific examples of dependency calls and how they will be converted